controller/userController.js

```diff
@@ -80,18 +74,16 @@ const token = (req, res) => {
                 return res.status(HttpStatus.OK).json(response);
             } else {
                 return res.status(HttpStatus.NOT_FOUND).json({
-                    status: StatusEnum['ERROR'],
                     message: 'User not found.'
                 });
             }
         }).catch(err => {
             return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
-                status: StatusEnum['ERROR'],
-                message: 'Internal server error.'
+                message: ValidationUtil.buildErrorMessage(err, 'token', 'user')
```

Comment:
Same as above comment.

Reply:
I am also now seeing that on lines 66 and 67 you are using the same object name `user`. I am not sure if that brings confusion as to which one it takes going forward. Maybe refactor by removing line 66 and just injection the body of the first `user` directly into `findOne`.

Reply:
I'll do that, but in the next commit don't bother to review this method because it will be implemented differently for sure. `tokenList` will be replaced with `jwt-redis` probably. We will discuss about how to store `refreshToken`